index.bs

```diff
@@ -743,6 +749,11 @@ The <dfn method for=CookieStore>delete(|options|)</dfn> method steps are:
         |options|["{{CookieStoreDeleteOptions/name}}"],
         |options|["{{CookieStoreDeleteOptions/domain}}"], and
         |options|["{{CookieStoreDeleteOptions/path}}"].
+    1. If the |options|["{{CookieStoreDeleteOptions/partitioned}}"] field is present and the user agent supports [cookie partitioning](https://github.com/WICG/CHIPS), then
```

Comment:
"field" isn't the right terminology.  Also, per previous comments, if we can default this it simplifies the logic.

Reply:
Done